include/expr/Parser.h

```diff
@@ -230,7 +230,8 @@ namespace expr {
     /// expressions.
     static Parser *Create(const std::string Name,
                           const llvm::MemoryBuffer *MB,
-                          ExprBuilder *Builder);
+                          ExprBuilder *Builder,
+			  bool QueryIndependence);
```

Comment:
Unfortunately, it is one of those cases in which clang-format re-formats the entire document. If you are ok with that, I can run clang-format anyway.